Clients/Xamarin.Interactive.Client/PropertyEditor/InteractivePropertyInfo.cs

```diff
@@ -193,9 +186,9 @@ static object UnpackValue (object value, InteractiveObjectEditor editor)
                 return UnpackValue (interactiveRepresentation, editor);
             case InteractiveEnumerable source:
                 var name = TypeHelper.GetCSharpTypeName (source.RepresentedType.Name);
-                return $"{name} - {source.Count} items";
+                return Catalog.GetString($"{name} - {source.Count} items");
```

Comment:
space before `(`

Reply:
For `Catalog` calls like this, do we need a comment param as well?

Reply:
This can't even be properly translated. We should teach `GetString` about `FormattableString`, but until then you'd need the call to be this:

```csharp
Catalog.Format (Catalog.GetString (
    "{0} - {1} items",
    comment: "{0} is a CLR type name; {1} is a count (integer) of items"),
    name,
    source.Count);
```

Annoying, I know 😄